docker/plausible/docker-compose.yml

```diff
@@ -0,0 +1,97 @@
+services:
+  sfusd_db:
+    image: postgres
+    restart: always
+    shm_size: 128mb
+    ports:
+      - 5432:5432
+    environment:
+      POSTGRES_PASSWORD: mhallwfwwas
```

Comment:
oops! I meant to change the password to supersecret to be consistent with the `.env` file.